interfaces/Portalicious/src/app/components/registration-page-layout/components/ignore-duplicates-dialog/ignore-duplication-dialog.component.html

```diff
@@ -0,0 +1,37 @@
+<app-confirmation-dialog
+  #ignoreDuplicationDialog
+  header="Ignore duplication"
+  i18n-header
+  headerIcon="pi pi-clone"
+  proceedLabel="Approve"
+  i18n-proceedLabel="@@generic-approve"
+  [mutation]="approveMutation"
+  [mutationData]="{ reason: updateReason() }"
+>
+  <p
+    i18n
+    class="mb-7"
+  >
+    You're about to mark
+    {{
+      duplicatesRegistrationIds().length === 2
+        ? 'both'
+        : duplicatesRegistrationIds().length
+    }}
+    registrations as unique. This means that they will no longer appear as
+    duplicates of each other.<br /><br />However, if either is found to be a
+    duplicate of other registrations, their duplication status will be
+    'Duplicate'.
+  </p>
+  <app-form-field-wrapper
+    label="Add a reason"
+    i18n-label
+  >
+    <input
+      pInputText
+      type="text"
+      [(ngModel)]="updateReason"
+      autocomplete="off"
```

Comment:
Would it be necessary to add a `[required]=true`-attribute here?
Or is that check already somewhere else?